extensions/ql-vscode/src/query-history/store/query-history-local-query-dto-mapper.ts

```diff
@@ -32,10 +32,13 @@ export function mapLocalQueryItemToDomainModel(
     localQuery.failureReason,
     localQuery.completedQuery &&
       mapCompletedQueryInfoToDomainModel(localQuery.completedQuery),
-    localQuery.evalLogLocation,
-    localQuery.evalLogSummaryLocation,
-    localQuery.jsonEvalLogSummaryLocation,
-    localQuery.evalLogSummarySymbolsLocation,
+    {
+      log: localQuery.evalLogLocation,
+      humanReadableSummary: localQuery.evalLogSummaryLocation,
+      jsonSummary: localQuery.jsonEvalLogSummaryLocation,
+      summarySymbols: localQuery.evalLogSummarySymbolsLocation,
+      endSummary: undefined,
+    },
```

Comment:
Can we set this to `undefined` instead when the `evalLogLocation` is `undefined`?